misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/model/JClass.java

```diff
@@ -177,4 +178,8 @@ public String getShortClassName() {
     public String getPackageName() {
         return packageName;
     }
+
+    public void setAccessModifier(String accessModifier) {
```

Comment:
Are we using this method anywhere?

Reply:
This is added to set the access modifier of a binding object that is to be generated. It is not used at the moment, but we intend on providing a flag to set it during the command execution. This will be implemented in the near future.